imap_processing/cdf/config/imap_idex_global_cdf_attrs.yaml

```
@@ -11,7 +11,7 @@ instrument_base: &instrument_base
 
 imap_idex_l1a_sci:
   <<: *instrument_base
-  Data_level: 1A
+  Data_level: L1A
```

```suggestion
  Data_level: 1A
```

I think this should stay as `1A` to be consistent with how the other instruments are labeling this?

---

Oops not sure why I changed that back.